plugins/elasticsearch/loggable-transport-netty4/src/main/java/org/opensearch/transport/LoggableNetty4ModulePlugin.java

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
It seems slightly weird to me that the path for these files is now `plugins/**elasticsearch**/loggable-transport-netty4/src/main/java/org/**opensearch**/transport/`. Is it weird to have elasticsearch and opensearch in the path? 

my intuition would be that we either stick entirely to `opensearch` or we do `elasticsearch` if we're building for an ES version and `opensearch` if we're building for an OS version, but I don't really get java namespacing, so maybe I'm missing something.

Reply:
Thanks for catching that, the org folder deep within the `plugins/elasticsearch` directory should've been elasticsearch but was missed after "forking" the plugin into two versions. I'll update it.